WISE-Community/WISE-Client
# Dynamic Prompt with Rules

## Changes

- Created Dynamic Prompt component
- Added Dynamic Prompt to Peer Chat
- Implemented Dynamic Prompt authoring
- Implemented Peer Grouping Different Ideas authoring
- Added Dynamic Prompt to Peer Chat grading view

## Test

Make sure to pull the latest from the WISE-API develop branch.

- Authoring Dynamic Prompt in Peer Chat
- Make sure the correct Dynamic Prompt shows up in the VLE for the student Peer Chat group
- Make sure the Dynamic Prompt shows up in the grading view

Closes #840
